lib/fastlane/plugin/clean_testflight_testers/actions/clean_testflight_testers_action.rb

```diff
@@ -9,33 +9,51 @@ def self.run(params)
         Spaceship::Tunes.select_team
         UI.message("Login successful")
 
-        UI.message("Fetching all TestFlight testers, this might take a few minutes, depending on the number of testers")
-
+        app_identifiers = params[:app_identifier].split(",").map(&:strip) if params[:app_identifier]
+        app_identifiers = Array(app_identifiers)
+        
         # Convert from bundle identifier to app ID
-        spaceship_app ||= Spaceship::Application.find(params[:app_identifier])
-        UI.user_error!("Couldn't find app '#{params[:app_identifier]}' on the account of '#{params[:username]}' on iTunes Connect") unless spaceship_app
-        app_id = spaceship_app.apple_id
+        apps = []
+        unless app_identifiers.any?
+          UI.message("Fetching all apps, this might take a few minutes, depending on the number of apps 🤖")
+          apps = Spaceship::Application.all
+        else
+          UI.message("Fetching the #{app_identifiers.size} apps, you specified, give me a moment 🤖")
+          app_identifiers.each do |app_identifier|
+            app = Spaceship::Application.find(app_identifier)
+            unless app
+              UI.message("Couldn't find app '#{app_identifier}' on the account of '#{params[:username]}' on iTunes Connect 😵")
+            else
+              apps << app
+            end
+          end
+          UI.user_error!("💥 Failed to find all apps specified 💥") unless app_identifiers.size == apps.size
+        end
 
-        all_testers = Spaceship::TestFlight::Tester.all(app_id: app_id)
         counter = 0
```

Comment:
We went from mere 4 testers being removed on our most popular test app to 164 when rolling over all apps 👍 Only took 27 seconds to go through 🤖 . Yet for a human.... 😱